projboard/tests/test_client.py

```diff
@@ -5,3 +5,29 @@
 def test_homepage(client):
     response = client.get("/")
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
```

Comment:
Do you need db here?
Please remove

Reply:
when I delete this mark the test failed with error
`..Database access not allowed, use the "django_db" mark...`

Reply:
OK, will have to understand, why
I choose not to die on that hill